examples/mag/README.md

```diff
@@ -188,3 +188,42 @@ python3 -m graphstorm.run.gs_node_classification \
 
 The accuracy of RGCN with the BERT model fine-tuned with venue prediction is 63.22%,
 while the accuracy of HGT is 67.20%.
+
+### Co-training BERT and GNN models using GLEM to predict the venue
+
+[GLEM](https://arxiv.org/abs/2210.14709) is a variational EM framework that trains a LM and GNN iteratively for semi-supervised node classification. There are two important pre-requisite for achieve good performance with GLEM
+
+1. The pseudolabeling technique: it predicts pseudolabels on the unlabeled nodes and uses as additional supervision signal for mutual distillation between LM and GNN. This can be enabled by the `--use-pseudolabel true` argument in command line. 
+2. Well pre-trained LM and GNN before the co-training: empirically, LM or GNN models that are not well-trained lead to degraded performance when co-training with GLEM directly. Therefore, we suggest user to pre-train the LM and GNN first. This can be achieved by:
+	1. Setting `num_pretrain_epochs` in the [yaml config](mag_glem_w_pretrain.yaml). 
```

Comment:
does `num_pretrain_epochs` train the GNN model? should we assume that users fine-tune the BERT model on the graph data first?

Reply:
`num_pretrain_epochs` train both the LM and GNN models separately (without co-training). We don't assume whether the BERT is fine-tuned or not.